controllers/upgrade_test.go

```diff
@@ -385,3 +386,48 @@ func TestDesiredNodesReady(t *testing.T) {
 		}
 	}
 }
+
+// Test
+func TestNodeTurnsOntoStep(t *testing.T) {
```

Comment:
Also this to `controllers/metrics_test.go`

Reply:
Moved to upgrade_metrics_test.go